### PR TITLE
change the current readme to the template

### PR DESCRIPTION
**Pull request by jbabazadeh**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10075

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 1]
```diff
@@ -2944,8 +3003,12 @@ Uploads indicators from Demisto to QRadar.
 #### Context Output
 
 There is no context output for this command.
-### qradar-ips-source-get
 
+</details>
+
+<details><summary><h3 style={{display: 'inline'}}>qradar-ips-source-get</h3></summary> 
+
+### qradar-ips-source-get
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3011]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 2]
```diff
@@ -2944,8 +3003,12 @@ Uploads indicators from Demisto to QRadar.
...
 ***
 Get Source IPs
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3013]:
```suggestion
Get Source IPs.
```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 3]
```diff
@@ -3010,9 +3073,11 @@ Get Source IPs
 >|---|---|---|---|---|---|---|---|---|---|
 >| 0 | 654 | 2023-07-26T14:31:44.753000+00:00 | 31 | 2023-07-26T15:31:06.386000+00:00 | 64 | 0 | Net-10-172-192.Net_192_168_0_0 | 14 | 1.1.1.1 |
 
+</details>
 
-### qradar-ips-local-destination-get
+<details><summary><h3 style={{display: 'inline'}}>qradar-ips-local-destination-get</h3></summary> 
 
+### qradar-ips-local-destination-get
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3080]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 4]
```diff
@@ -3051,9 +3116,11 @@ Get Source IPs
 >### Local Destination IPs
 >**No entries.**
 
+</details>
 
-### qradar-search-retrieve-events
+<details><summary><h3 style={{display: 'inline'}}>qradar-search-retrieve-events</h3></summary> 
 
+### qradar-search-retrieve-events
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3123]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 5]
```diff
@@ -3102,8 +3169,11 @@ Polling command to search for events of a specific offense.
 
 >Search ID: 9c2c18a8-5e06-4edb-bc26-53ad44421148
 
-### qradar-remote-network-cidr-create
+</details>
+
+<details><summary><h3 style={{display: 'inline'}}>qradar-remote-network-cidr-create</h3></summary> 
 
+### qradar-remote-network-cidr-create
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3176]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 6]
```diff
@@ -3125,8 +3195,12 @@ Create remote network CIDRs.
 #### Context Output
 
 There is no context output for this command.
-### qradar-remote-network-cidr-list
 
+</details>
+
+<details><summary><h3 style={{display: 'inline'}}>qradar-remote-network-cidr-list</h3></summary> 
+
+### qradar-remote-network-cidr-list
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3203]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 7]
```diff
@@ -3156,8 +3230,11 @@ Retrieves the list of staged remote networks.
 | QRadar.RemoteNetworkCIDR.name | String | The associated CIDR name as it appears in QRadar. | 
 | QRadar.RemoteNetworkCIDR.description | String | The associated CIDR description as it appears in QRadar. | 
 
-### qradar-remote-network-cidr-delete
+</details>
+
+<details><summary><h3 style={{display: 'inline'}}>qradar-remote-network-cidr-delete</h3></summary> 
 
+### qradar-remote-network-cidr-delete
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3237]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 8]
```diff
@@ -3174,8 +3251,12 @@ Deletes an existing staged remote network.
 #### Context Output
 
 There is no context output for this command.
-### qradar-remote-network-cidr-update
 
+</details>
+
+<details><summary><h3 style={{display: 'inline'}}>qradar-remote-network-cidr-update</h3></summary> 
+
+### qradar-remote-network-cidr-update
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3259]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 9]
```diff
@@ -3205,8 +3286,11 @@ Updates an existing staged remote network.
 | QRadar.RemoteNetworkCIDR.group | String | The group to which the remote network belongs. | 
 | QRadar.RemoteNetworkCIDR.description | String | The description of the remote network. | 
 
-### qradar-remote-network-deploy-execution
+</details>
 
+<details><summary><h3 style={{display: 'inline'}}>qradar-remote-network-deploy-execution</h3></summary> 
+
+### qradar-remote-network-deploy-execution
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3293]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 10]
```diff
@@ -3229,8 +3313,12 @@ Potentially harmful: This API command executes any waiting system deployments in
 | **Path** | **Type** | **Description** |
 | --- | --- | --- |
 | QRadar.deploy.status | String | The deployment status \(INITIALIZING, IN_PROGRESS, COMPLETE\). | 
-### qradar-log-source-extensions-list
 
+</details>
+
+<details><summary><h3 style={{display: 'inline'}}>qradar-log-source-extensions-list</h3></summary> 
+
+### qradar-log-source-extensions-list
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3321]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 11]
```diff
@@ -3256,8 +3344,11 @@ Retrieves a list of log source extensions.
 | QRadar.LogSourceExtension.ID | Number | The ID of the extension. | 
 | QRadar.LogSourceExtension.UUID | String | The UUID string of the log source extension. | 
 
-### qradar-log-source-delete
+</details>
+
+<details><summary><h3 style={{display: 'inline'}}>qradar-log-source-delete</h3></summary> 
 
+### qradar-log-source-delete
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3351]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 12]
```diff
@@ -3389,8 +3489,11 @@ Retrieves a list of log source languages.
 | QRadar.LogSourceLanguage.ID | Number | The ID of the language. This ID does not change across deployments. | 
 | QRadar.LogSourceLanguage.Name | String | The display name of the language. | 
 
-### qradar-log-source-protocol-types-list
+</details>
 
+<details><summary><h3 style={{display: 'inline'}}>qradar-log-source-protocol-types-list</h3></summary> 
+
+### qradar-log-source-protocol-types-list
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3496]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 13]
```diff
@@ -3308,8 +3402,11 @@ In order to get wincollect_external_destination_ids - filter internal=false need
 | QRadar.WinCollectDestination.IsInternal | Boolean | Set to "true" if the destination corresponds to a QRadar event collector process from this deployment; otherwise, it is set to false if it is any other host. | 
 | QRadar.WinCollectDestination.EventRateThrottle | Number | The events-per-second rate that is used to throttle the event flow to this destination. | 
 
-### qradar-log-source-create
+</details>
 
+<details><summary><h3 style={{display: 'inline'}}>qradar-log-source-create</h3></summary> 
+
+### qradar-log-source-create
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3409]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 14]
```diff
@@ -3364,8 +3461,11 @@ Creates a new log source.
 | QRadar.LogSource.Status | unknown | Status of the log source. | 
 | QRadar.LogSource.TargetEventCollectorID | Number | The ID of the event collector where the log source sends its data. | 
 
-### qradar-log-source-languages-list
+</details>
+
+<details><summary><h3 style={{display: 'inline'}}>qradar-log-source-languages-list</h3></summary> 
 
+### qradar-log-source-languages-list
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3468]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 15]
```diff
@@ -3445,8 +3548,11 @@ Retrieves the list of protocol types.
 | QRadar.LogSourceProtocolType.TestingCapabilities.testable | Boolean | Indicates whether the protocol type is testable. | 
 | QRadar.LogSourceProtocolType.Version | String | The version of the protocol type component. | 
 
-### qradar-disconnected-log-collectors-list
+</details>
+
+<details><summary><h3 style={{display: 'inline'}}>qradar-disconnected-log-collectors-list</h3></summary> 
 
+### qradar-disconnected-log-collectors-list
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3555]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 16]
```diff
@@ -3474,8 +3580,11 @@ Retrieves a list of disconnected log collectors.
 | QRadar.DisconnectedLogCollector.UUID | String | The UUID of the disconnected log collector. | 
 | QRadar.DisconnectedLogCollector.Version | String | The version of the disconnected log collector. | 
 
-### qradar-log-source-update
+</details>
 
+<details><summary><h3 style={{display: 'inline'}}>qradar-log-source-update</h3></summary> 
+
+### qradar-log-source-update
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3587]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 17]
```diff
@@ -3511,8 +3620,12 @@ Updates an exising log source.
 #### Context Output
 
 There is no context output for this command.
-### qradar-log-source-types-list
 
+</details>
+
+<details><summary><h3 style={{ display: 'inline' }}>qradar-log-source-types-list</h3></summary> 
+
+### qradar-log-source-types-list
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3628]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 18]
```diff
@@ -3546,8 +3659,12 @@ Retrieves a list of log sources types.
 | QRadar.LogSourceTypesList.uuid | String | A UUID string of the log source type. | 
 | QRadar.LogSourceTypesList.version | String | The log source type plugin version. | 
 
-### qradar-log-source-groups-list
+</details>
+
 
+<details><summary><h3 style={{display: 'inline'}}>qradar-log-source-groups-list</h3></summary> 
+
+### qradar-log-source-groups-list
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3667]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 19]
```diff
@@ -3577,8 +3694,11 @@ Retrieves a list of log source languages.
 | QRadar.LogSourceGroup.Owner | String | The name of the user who owns the group. | 
 | QRadar.LogSourceGroup.ParentID | Number | The ID of the group's parent group. The root node group has a null parent_ID. | 
 
-### qradar-event-collectors-list
+</details>
 
+<details><summary><h3 style={{display: 'inline'}}>qardar-event-collectors-list</h3></summary> 
+
+### qardar-event-collectors-list
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3701]:
```suggestion

```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 20]
```diff
@@ -3604,3 +3724,33 @@ Retrieves a list of event collectors.
...
 | QRadar.EventCollector.ID | Number | The unique ID of the event collector. | 
 
+</details>
+
+
+
+# Troubleshooting
+<details> <summary><h3 style={{display: 'inline'}}>Instance Configuration</h3></summary> 
+
+To enable the execution of commands using QRadar on CLOUD, it is essential to add the XSOAR IPs to the QRadar whitelist. You can locate the XSOAR IP by referring to this [link](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Administrator-Guide/Enable-Access-to-Cortex-XSOAR) under the **"Used for communication between Cortex XSOAR and customer resources"** section, depending on your tenant region.
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3734]:
```suggestion
To enable the execution of commands using QRadar on CLOUD, it is essential to add the XSOAR IPs to the QRadar whitelist. You can locate the Cortex XSOAR IP by referring to this [link](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Administrator-Guide/Enable-Access-to-Cortex-XSOAR) under the **"Used for communication between Cortex XSOAR and customer resources"** section, depending on your tenant region.
```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 21]
```diff
@@ -3604,3 +3724,33 @@ Retrieves a list of event collectors.
...
+```
+It is recommended to enable [mirroring](#mirroring-events), as it should fetch previously missed events when the offense is updated.
+Alternatively, the [retrieve events command](#qradar-search-retrieve-events) can be used to retrieve the `events` immediately.
+If the command takes too long to finish executing, try setting the `interval_in_seconds` to a lower value (down to a minimum of 10 seconds).
+</details>
+
+<details> <summary><h3 style={{display: 'inline'}}> General </h3> </summary> 
+
+If you encounter a 403 Forbidden error while attempting to execute API calls, start by verifying your QRadar permissions and ensuring that you have the appropriate access based on [those permissions](#Permissions).
+If the issue persists, access your QRadar instance with the same user you are using on XSOAR and navigate to the Interactive API for Developers page (instructions [here](https://www.ibm.com/docs/en/qradar-on-cloud?topic=api-accessing-interactive-documentation-page)). Try running the request from there, using the relevant endpoint with the intended arguments. If you still receive a 403 error, please contact your QRadar support.
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:3754]:
```suggestion
If the issue persists, access your QRadar instance with the same user you are using in Cortex XSOAR and navigate to the Interactive API for Developers page (instructions [here](https://www.ibm.com/docs/en/qradar-on-cloud?topic=api-accessing-interactive-documentation-page)). Try running the request from there, using the relevant endpoint with the intended arguments. If you still receive a 403 error, contact your QRadar support.
```

**Comment by ShirleyDenkberg** [comment]
@jbabazadeh Doc review completed.

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 22]
```diff
@@ -87,63 +53,98 @@ Every command and playbook that runs in QRadar v2 also runs in QRadar v3. No adj
...
-Nevertheless, some events may not be available when trying to fetch them during an incident creation. If **Retry events fetch** is enabled, the integration tries to fetch more events when the number fetched is less than the expected `event_count`. In the default setting, the integration will try 3 times, with a wait time of 100 seconds between retries.
-In order to change the default values, configure the following **Advanced Parameters** in the instance configuration:
-```
-EVENTS_SEARCH_TRIES=<amount of tries for events search> (default 3),EVENTS_SEARCH_RETRY_SECONDS=<amount of seconds to wait between tries> (default 100),EVENTS_POLLING_TRIES=<number of times to poll for one search> (default 10),
-```
-It is recommended to enable [mirroring](#mirroring-events), as it should fetch previously missed events when the offense is updated.
-Alternatively, the [retrieve events command](#qradar-search-retrieve-events) can be used to retrieve the `events` immediately.
-If the command takes too long to finish executing, try setting the `interval_in_seconds` to a lower value (down to a minimum of 10 seconds).
+# Permissions
```
[the remark this answers is not in the document]
Comment (jbabazadeh) [reply Packs/QRadar/Integrations/QRadar_v3/README.md:71]: no we are working now to change the template.

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 23]
```diff
@@ -87,63 +53,98 @@ Every command and playbook that runs in QRadar v2 also runs in QRadar v3. No adj
...
+| --- | --- |
+| Assets | Vulnerability Management *or* Assets |
+| Domains | Admin |
+| Offenses (Manage Closing Reason) | Manage Offense Closing Reasons |
+| Offenses (Assign Offenses to Users) | Assign Offenses to Users |
+| Offenses (Read) | Offenses |
+| References (Create/Update) | Admin |
+| References (Read) | View Reference Data |
+
+# Configure Integration on Cortex XSOAR
```
[the remark this answers is not in the document]
Comment (jbabazadeh) [reply Packs/QRadar/Integrations/QRadar_v3/README.md:82]: same here

**Pull request closed without merging**